book/reqs.go

```diff
@@ -137,7 +137,9 @@ func CheckRequirements() (bool, error) {
 	}
 
 	// Docker Compose
-	if _, err := exec.LookPath("docker-compose"); err != nil {
+	_, composeBinary := exec.LookPath("docker-compose")
+	composePlugin := exec.Command("docker", "compose").Run()
```

Comment:
Can't we check the version of the `docker` binary instead?

Reply:
Presence of the docker binary does not mean you have the `compose` subcommand.

Reply:
Oh yeah, but you can have the latest Docker CLI and no compose plugin?

Reply:
Indeed. And docker compose is still available as a standalone binary as well.

Reply:
oh, right, this is a plugin.